### DISCUSSION TITLE
Plastic material and Damage

### DISCUSSION THREAD

**Discussion by guangdongzhang** [peridigm/peridigm#293] [Q&A]
Hello everyone,
          Can both the plastic material model and damage criteria be setted in one modleing in Peridigm?

**Comment by jritta** [comment]
Hello @guangdongzhang,

Yes you can set the material model and damage model in the input deck (preferred in yaml format) independently. But be aware that some (maybe even most) combinations might not really make sense from a physical point of view.

Best,
Jonas
